ui/app/models/kv/data.js

```diff
@@ -44,7 +44,7 @@ export default class KvSecretDataModel extends Model {
   path;
 
   @attr('object')
-  data;
```

Comment:
`data` was being normalized out of the payload, so I renamed it so the data is preserved